pkgs/applications/misc/protonup-qt/default.nix

```diff
@@ -0,0 +1,21 @@
+{ appimageTools, fetchurl, lib }:
+let
+  pname = "protonup-qt-bin";
+  version = "2.7.4";
+in
+appimageTools.wrapType2 rec {
+  name = "protonup-qt";
```

Comment:
should have pname and version here rather than just name

Reply:
fixed

Reply:
Sorry I meant it should be only pname and version, not as well as name.

`name` is computed from `pname` and `version` together, so there's no need to set both.

Reply:
It seems when using an appimage wrapper, `name` sets the name of the binary produced. Getting rid of the name variable will cause the binary to be called `protonup-qt-bin-2.7.4`.